_posts/2022-05-04-RTK-Intership.md

```diff
@@ -0,0 +1,249 @@
+The ANN-MB-00 GNSS multiband antenna is unique from other GNSS/GPS antennas in that it is designed to receive both the classic L1 GPS band and the fairly newly launched L2 GPS band. 
+Furthermore, the u-blox ANN-MB-00 is well constructed with a magnetic base with mounting holes for additional anchoring for the toughest environments.
+It is suitable for use with any GPS/GNSS receiver with dual L1/L2 reception and supports GPS, GLONASS, Galileo and BeiDou.
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/antenna.jpg' | prepend: site.baseurl }}" alt="ANN-MB-00 antenna" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+### Testing of the antennas
+I tested both antennas side by side for 5 minutes while outside on the table, using SBAS as the correction method.
+These are the results (the more compact the data points the more accurate):
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/TestingAntennasOutside.JPG' | prepend: site.baseurl }}" alt="TestingAntennasOutside" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+
+u-blox ANN-MB-00-00 (new antenne):
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/newantenna.png' | prepend: site.baseurl }}" alt="newantenna" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+You can see that the antenna stays within a range of max 2.5m when stationary. 
+It got a connection to the signal immediately when I started the module.
+
+Unknown first cheap antenna:
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/Cheapantenna.png' | prepend: site.baseurl }}" alt="Cheapantenna" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+First of all it took way longer to establish a connection, a couple of minutes. 
+The data points are further apart from each other. 
+Whereas you can see with the other antenna that they are much closer together. 
+So that means that there is a lot of play between each data point. 
+This means that the new antenna is more accurate. 
+At last you can also see that the range of the antenna went over the 2.5 meter mark.
+
+
+From this we can conclude that it is necessary to have a good antenna when we want to work precisely.
+
+### Testing with RTK
+There were a couple of problems with the SparkFun GPS-RTK Board during my internship, it didn't calculate the accurate positioning altho it did receive the correction data correctly from FLEPOS.
+The Sparkfun RTK-board is supposed to calculate this automatically, but it didn't and I could not change the configuration to the right settings.
+That is why I don't have a video which shows the use of RTK on the SparkFun GPS-RTK Board like in the section with SBAS.
+
+
+## Sending data to the cloud
+As an extra you can send the data to the cloud, during my internship at Ordina I did this as follows:
```

Comment:
If you have more information about this, please expand it. If not, this is fine.